### DISCUSSION TITLE
ender 3 V2

### DISCUSSION THREAD

**Discussion by scooby915** [mriscoc/Ender3V2S1#1215] [General]
is there a way to change the points that the tram feature and auto build mesh use? I am using on a larger printer and the stop points are off even after changing the physical bed size in the FW.

**Comment by TheWebMachine** [comment]
You may need to adjust the X and Y Min values. The bed is a bit larger than the stated build volume. So your X and Y Min values should be a negative value, such that your Origin (0x, 0y) will be inset from the corner of the bed. Your Max values will also be a bit higher than the build volume, too. 

For example, on my S1 V2 Plus (build volume 300³), the values measured out as follows:
X Bed = 300
Y Bed = 300
X Min = -8
Y Min = -3
X Max = 323
Y Max = 310
Z Max = 310

This means that my X axis actually has the ability to move 331mm in total and Y can move 318mm. I arrived at true Min and Max by setting Max on all axis to 350mm (to allow me to live move without arbitrary stops) and manually moving the nozzle to direct center of the bed, then using live move to determine the distance to the axis endstop, subtract 150 (half of bed dimension), the rest is margin and should be accounted for in Min values by way of a negative value.

You definitely want that margin in negative Min values, as your slicer wants to drop the nozzle purge/clean line in that margin.

Also keep in mind with setting the Mesh size, if you don't use the Center Area option (which accounts for probe offset and subtracts from all sides), the mesh and tram points will appear shifted to account for that probe offset.

Ex. My printer:
Max Area Xmin, Xmax, Ymin, Ymax = 10, 277.6, 10, 290
Centered Xmin, Xmax, Ymin, Ymax = 22.4, 277.6, 22.4, 277.6

As you can see, Max Area will draw the equal mesh regions off-center from the bed's true center. Centered will put it proper centered to the bed's true center...if you have proper Physical Min/Max values.

Does that make sense?

**Comment by mriscoc** [comment]
Use Advanced / Mesh Leveling / Mesh Inset / `Maximize Area` that will set the mesh leveling area to the maximum reachable positions by the probe.

**Comment by scooby915** [comment]
Yes it does but I have been trying to get the setting right and still seem to be struggling. I think I understand now after rereading your response and will give it a shot tonight. Last night I  got the home to center correctly and tram correct but it still wanted to print off center and my purge line was still off the bed. I am trying to set up a ender 3 Neo max with a 300 x 300 bed. I am sure I am just overthinking it.

On Jan 13, 2024, at 9:11 PM, TheWebMachine ***@***.***> wrote:

﻿

You may need to adjust the X and Y Min values. The bed is a bit larger than the stated build volume. So your X and Y Min values should be a negative value, such that your Origin (0x, 0y) will be inset from the corner of the bed. Your Max values will also be a bit higher than the build volume, too.

For example, on my S1 V2 Plus (build volume 300³), the values measured out as follows:
X Bed = 300
Y Bed = 300
X Min = -8
Y Min = -3
X Max = 323
Y Max = 310
Z Max = 310

This means that my X axis actually has the ability to move 331mm in total and Y can move 318mm. I arrived at true Min and Max by setting Max on all axis to 350mm (to allow me to live move without arbitrary stops) and manually moving the nozzle to direct center of the bed, then using live move to determine the distance to the axis endstop, subtract 150 (half of bed dimension), the rest is margin and should be accounted for in Min values by way of a negative value.

You definitely want that margin in negative Min values, as your slicer wants to drop the nozzle purge/clean line in that margin.

Also keep in mind with setting the Mesh size, if you don't use the Center Area option (which accounts for probe offset and subtracts from all sides), the mesh and tram points will appear shifted to account for that probe offset.

Ex. My printer:
Max Area Xmin, Xmax, Ymin, Ymax = 10, 277.6, 10, 290
Centered Xmin, Xmax, Ymin, Ymax = 22.4, 277.6, 22.4, 277.6

As you can see, Max Area will draw the equal mesh regions off-center from the bed's true center. Centered will put it proper centered to the bed's true center...if you have proper Physical Min/Max values.

Does that make sense?
